src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -168,6 +169,7 @@ public abstract partial class JsonNamingPolicy
         public bool NameEquals(System.ReadOnlySpan<char> text) { throw null; }
         public bool NameEquals(string text) { throw null; }
         public override string ToString() { throw null; }
+        public void Write(System.Text.Json.Utf8JsonWriter writer) { }
```

Comment:
While I agree both WriteProperty (because it won't ask for the name) and WriteValue (because it's writing a property) are wrong here, Write feels... wrong.  WriteTo seems better; but is then inconsistent with other things on these types.

Maybe it's better to take this as a known area of improvement instead of making a naming decision with no runway for change.

Reply:
Alright. Removing both new APIs being added.